shell/key-bindings.fish

```diff
@@ -1,7 +1,9 @@
 # Key bindings
 # ------------
 function fzf_key_bindings
-
+  set -g __FZF_FISH_MAJOR (echo $FISH_VERSION | cut -f1 -d.)
```

Comment:
How about just doing this inside `fzf-history-widget`? We can create local variables instead only when they're needed, which makes us repeat it whenever the widget's called, but it won't cause any noticeable delay.